tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "client/src/**/*",
     "client/utils/**/*",
     "tools/challenge-helper-scripts/**/*",
+    "tools/scripts/lint/validate-keys.ts",
```

Comment:
```suggestion
```
Sorry, @foolhardy21 I forgot that the relevant `tsconfig.json` is inside `tools/scripts/build`.  The thing to do is *probably*, move the tsconfig to `tools/` and update the scripts accordingly.

Reply:
hey @ojeytonwilliams, I have moved the `tsconfig.json` from `tools/scripts/build` to `tools` and also updated the `npm run validate-keys` in `client/package.json`. Due to this, I had to update the project in `.eslintrc.json` and the `lint:ts` testing script in the root 'package.json`. I have added `validate-keys.js` to `.gitignore` and `.eslintignore`. Let me know about further updates. thanks.